### DISCUSSION TITLE
Feature request: Show IIS eventlog in the portal

### DISCUSSION THREAD

**Discussion by mwiseley-wakefly** [umbraco/Umbraco.Cloud.Issues#833] [General]
I did a recent deployment to Umbraco Cloud with a Production web.config transformation that resulted in a validation error in the web.config, taking down the Live site. The site served 500 status without any detail, and there was nothing indicating the issue in either the Umbraco trace logs via Kudu or the log messages shown in the Cloud dashboard. On a server, IIS reports an issue like this in the Windows Event Viewer Application log. It would avoid a lot of guesswork to have visibility into this kind of error in Umbraco Cloud.

**Comment by jemayn** [comment]
I think this is a good idea, and could be added to the /project/{projectalias}/logs/{environmentAlias} page.

Until a day where that is prioritized I just want to say that the event log can be found in Kudu:

```
C:\home\LogFiles\eventlog.xml
```

But a better view of that without requiring going to Kudu would be much prefferred.

We have a duplicate feature request here: Duplicate discussion here: https://github.com/umbraco/Umbraco.Cloud.Issues/discussions/899

**Comment by jemayn** [comment]
Hey @mwiseley-wakefly 

This feature has now been added, if you go to the log page via the context menu on your environment, or go directly via
> /project/{project-alias}/logs/{environment-alias}#eventlog

Because of that I will close this feature request. Thanks for taking the time to help us improve the platform! 🙂
